styles/components/alert.css

```diff
@@ -0,0 +1,35 @@
+.alert {
+  background: rgba(255, 255, 255, 0.3);
+  backdrop-filter: blur(6.5px);
+  -webkit-backdrop-filter: blur(6.5px);
+  padding: 15px 20px;
```

Comment:
improvement - change px to rem, as they are responsive font unit

Reply:
ah okay i see